docs/getting_started/basics/debug.md

```diff
@@ -7,7 +7,10 @@ While working on a workflow, debugging may be necessary to ensure the graph exec
 
 When a node does not execute successfully it will turn red and the workflow execution will be interrupted. In such case, the node will store the traceback generated by the error. To inspect the traceback, `Right Click` on the node and select the *Traceback* option. This will pop up a dialog with the error traceback content. 
 
-![Traceback](../../images/traceback.gif)
+<figure>
+      <img src="images/traceback.gif" alt="Traceback">

```

Comment:
In this gif still appears the old "Operator Name" naming in the inspector, should we update it?

Reply:
I guess we have to update them all 😢

Reply:
Changed 
https://github.com/Inbibo/shift_docs/pull/61/commits/69e91498ee0d9a270c8014ac34740c78c4906515

Reply:
I think this one has not been updated.

Reply:
Now is updated